mule-extensions-api/src/main/java/org/mule/runtime/extension/api/declaration/type/annotation/ConfigOverrideTypeAnnotation.java

```diff
@@ -29,10 +29,10 @@ public String getName() {
   /**
    * {@inheritDoc}
    * 
-   * @return {@code false}
+   * @return {@code true}
    */
   @Override
   public boolean isPublic() {
-    return false;
+    return true;
```

Comment:
doesn't this require you to make changes in the persistence module as well? LIke adding a name mapping, updating the tests, etc?